llvm/labs/lab2/zakharov_artem/mtbs.h

```diff
@@ -0,0 +1,17 @@
+#ifndef LLVM_MTBS_H
```

Comment:
More generic solution is to move main pass class definition into anonymous namespace in cpp file and do not use header file.